brambling/views/organizer.py

```diff
@@ -1298,11 +1298,13 @@ def get_forms(self):
                         break
         self.transaction_refund_forms = [TransactionRefundForm(t)
                                          for t in self.order.transactions.all()]
+
+        # Transaction refund forms are not handled by this view so don't
+        # get included here.
+        forms = [self.notes_form] + self.attendee_forms
         if can_edit_event:
-            forms = [self.payment_form, self.notes_form, self.transaction_refund_forms]
-        else:
-            forms = [self.notes_form]
-        return forms + self.attendee_forms
+            forms.append(self.payment_form)
```

Comment:
This is arguably a silly way of doing this but redoing it would require rewriting the view for (at this point) little benefit IMO.